tests/pytorch/nightly/dlrm.libsonnet

```diff
@@ -171,14 +171,31 @@ local utils = import 'templates/utils.libsonnet';
     ),
   },
 
+  local tpuVm = common.PyTorchTpuVmMixin {
+    tpuSettings+: {
+      tpuVmExtraSetup: |||
+        pip3 install tqdm sklearn tensorboardX google-cloud-storage
+        git clone -b tpu-xrt --single-branch https://github.com/darisoy/dlrm.git dlrm-xrt/
+        echo 'export PATH=~/.local/bin:$PATH' >> ~/.bash_profile
+      |||,
+    },
+    paramsOverride+: {
+      scriptPath: 'dlrm-xrt/dlrm_tpu_runner.py',
+    },
+  },
+
   local v3_8 = {
     accelerator: tpus.v3_8,
   },
+  local v4_8 = {
+    accelerator: tpus.v4_8,
+  },
   configs: [
     dlrm + v3_8 + one_core + timeouts.Hours(3) + mixins.Experimental,
     dlrm + v3_8 + seq_fwd + timeouts.Hours(3) + mixins.Experimental,
     dlrm + v3_8 + mp_fwd + timeouts.Hours(3) + mixins.Experimental,
     dlrm + v3_8 + mp_dp_fwd + timeouts.Hours(3),
     dlrm + v3_8 + criteo_kaggle + timeouts.Hours(6),
+    dlrm + v4_8 + criteo_kaggle + timeouts.Hours(6) + tpuVm,
```

Comment:
FYI, you may need to adjust the timeout here since you're running it for 25 epochs instead of 1